### PR TITLE
load_sample: add a clear error message for datasets missing website registration

### PR DESCRIPTION
**Pull request by neutrinoceros**
## PR Summary

got the opportunity to catch and fix this while testing #3258

### CODE REVIEW

File: yt/loaders.py [hunk 1]
```diff
@@ -1341,6 +1341,14 @@ def load_sample(fn=None, progressbar: bool = True, timeout=None, **kwargs):
             "Registry appears to be corrupted: could not find a 'load_name' entry for this dataset."
         )
 
+    if not isinstance(specs["load_kwargs"], dict):
+        raise ValueError(
+            "The requested dataset seems to be improperly registered.\n"
+            "Tip: the entry in yt/sample_data_registry.json may inconstistent with "
```
Comment (neutrinoceros) [inline yt/loaders.py:1347]:
```suggestion
            "Tip: the entry in yt/sample_data_registry.json may be inconststent with "
```

File: yt/loaders.py [hunk 2]
```diff
@@ -1344,7 +1344,7 @@ def load_sample(fn=None, progressbar: bool = True, timeout=None, **kwargs):
     if not isinstance(specs["load_kwargs"], dict):
         raise ValueError(
             "The requested dataset seems to be improperly registered.\n"
-            "Tip: the entry in yt/sample_data_registry.json may inconstistent with "
+            "Tip: the entry in yt/sample_data_registry.json may be inconststent with "
```
Comment (chummels) [inline yt/loaders.py:1347]:
```suggestion
            "Tip: the entry in yt/sample_data_registry.json may be inconsistent with "
```

Comment (neutrinoceros) [reply yt/loaders.py]: Damn how can I get this wrong _twice_ ? Thanks for picking it

**Review by cphyc — APPROVED** [review]

**Review by chummels — APPROVED** [review]
Looks good!

**Pull request merged**